lib/system_theme.dart

```diff
@@ -84,17 +84,21 @@ class SystemAccentColor {
   /// Updates the fetched accent colors on Windows.
   Future<void> load() async {
     WidgetsFlutterBinding.ensureInitialized();
-
-    final colors = await _channel.invokeMethod(kGetSystemAccentColorMethod);
-    if (colors == null) return;
-
-    accent = _retrieve(colors['accent'])!;
-    light = _retrieve(colors['light']) ?? accent;
-    lighter = _retrieve(colors['lighter']) ?? accent;
-    lightest = _retrieve(colors['lightest']) ?? accent;
-    dark = _retrieve(colors['dark']) ?? accent;
-    darker = _retrieve(colors['darker']) ?? accent;
-    darkest = _retrieve(colors['darkest']) ?? accent;
+    
+    try {
+      final colors = await _channel.invokeMethod(kGetSystemAccentColorMethod);
+      if (colors == null) return;
+
+      accent = _retrieve(colors['accent'])!;
+      light = _retrieve(colors['light']) ?? accent;
+      lighter = _retrieve(colors['lighter']) ?? accent;
+      lightest = _retrieve(colors['lightest']) ?? accent;
+      dark = _retrieve(colors['dark']) ?? accent;
+      darker = _retrieve(colors['darker']) ?? accent;
+      darkest = _retrieve(colors['darkest']) ?? accent;
+    } catch (e) {
+      return;
+    }
```

Comment:
I think only platform exceptions should be handled. What do you think?

```suggestion
    } on PlatformException catch (e) {
      return;
    } catch (e) {
      rethrow;
    }
```

Reply:
```suggestion
    } on MissingPluginException catch (e) {
      return;
    } catch (e) {
      rethrow;
    }
```
It's weird that I found that this lib not implements iOS platform plugin. That's why #18 happened: Flutter tries to load platform channel but there is none. So `MissingPluginException` should be handled instead.

I was confused with iOS support in `README.md`, while it is actually unsupported in `pubspec.yaml`

Reply:
As it seems like, iOS doesn't have a native method that refers to the system colors - and I don't even think there is such thing as system theme on iOS. iOS support on README refers to the support to the `isDarkMode` implementation.

I still think it is valid to check if the system is iOS and do not call the platform method